### ISSUE TITLE
Table is missing `display: flex` for the row and header containers

### ISSUE THREAD

**Issue by kaiyoma** [bvaughn/react-virtualized#893]
I'm trying to create a table with automatically sized columns that fill the full width of the table, so I'm setting `flexGrow={1}` on each column. However, this doesn't work out of the box. After banging my head on the table for a while and diving into the DOM, I realized that the row and header containers don't specify `display: flex`, so the `<div>` children get rendered as full-width, which is clearly wrong.

In short, I have to add this CSS to get the table to work correctly:

```css
.ReactVirtualized__Table {
  .ReactVirtualized__Table__headerRow,
  .ReactVirtualized__Table__row {
    display: flex;
  }
}
```

I'm assuming this is an oversight/bug, as I don't see this caveat mentioned anywhere in the docs.

**Comment by sajinshrestha** [comment]
Are you passing the required `width` (flex basis) property? 

```
<Column
  dataKey={column}
  width={200}
  flexGrow={1}
  ....
/>
```

Also, if you are importing the default style `import 'react-virtualized/styles.css'`, [display: flex](https://github.com/bvaughn/react-virtualized/blob/master/source/styles.css#L25-L36) should be set for you already.

**Comment by kaiyoma** [comment]
Ah yes, I didn't realize I had to import a stylesheet as well. Thanks!
